lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -61,7 +61,7 @@ namespace Js
         // in practice the modulerecord lifetime should be the same as the scriptcontext so it could be retrieved for the same
         // site. Host might hold a reference to the module as well after initializing the module.
         // In our implementation, we'll use the moduleId in bytecode to identify the module.
-        childModuleRecord->moduleId = scriptContext->GetLibrary()->EnsureModuleRecordList()->Add(childModuleRecord);
+        childModuleRecord->moduleId = scriptContext->GetLibrary()->EnsureModuleRecordList()->Add(childModuleRecord) + 1; // 0 == kmodGlobal
```

Comment:
Found a better way to check module code so this shift is no longer needed.

---
In reply to: [114905856](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114905856) [](ancestors = 114905856)